packages/create-sitecore-jss/src/common/Initializer.ts

```diff
@@ -1,9 +1,10 @@
 import { BaseArgs } from './args/base';
-
+import { Prerender } from '../initializers/nextjs/prompts';
 export type InitializerResults = {
   appName: string;
   initializers?: string[];
   nextSteps?: string[];
+  prerender?: Prerender;
```

Comment:
The same issue here, `prerender` can't be returned by the each framework, only related to the Nextjs.
Probably, better to add `{ [key: string]: unknown }`